### ISSUE TITLE
CIS health check resource creates resource that ignores certificates verification  

### ISSUE THREAD

**Issue by ashishth09** [IBM-Cloud/terraform-provider-ibm#1310]
<img width="560" alt="image" src="https://user-images.githubusercontent.com/11552451/79298099-b5ca9680-7efd-11ea-873e-c6cefa470b48.png">

```
resource "ibm_cis_healthcheck" "test" {
  cis_id         = data.ibm_cis_domain.domain.cis_id
  expected_body  = ""
  expected_codes = "200"
  method         = "GET"
  path           = var.waf_health_endpoint
  interval       = 60
  type           = "https"
  retries        = 5
  description    = "Health check for ${var.waf_health_endpoint}"
}
```

Happens with https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.2.5.

Looks like the default value of allow_insecure is true which is not right per me. I have raised the issue with CIS folks.
Can you please update the documentation at https://cloud.ibm.com/docs/terraform?topic=terraform-cis-resources#cis-health. It doesn't describe https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/ibm/resource_ibm_cis_healthcheck.go#L82 describing it in more details.

**Comment by hkantare** [comment]
We updated the docs yesterday https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/website/docs/r/cis_healthcheck.html.markdown
The cloud docs will be updated once we make a release

**Comment by ashishth09** [comment]
I think you should set that field's default value to false to comply with https://api.cloudflare.com/#load-balancer-monitors-create-monitor and avoid any confusion.

**Comment by hkantare** [comment]
https://cloud.ibm.com/apidocs/cis/glb-monitor#create-a-load-balancer-monitor...based on this API docs we don't find any default value

**Comment by ashishth09** [comment]
It should be a defect against them as there is definitely a default value of false. I talked to the CIS folks in internal slack channel.

**Comment by kavya498** [comment]
Available in
[](https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.4.0)
[](https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v0.26.0)

**Comment by kavya498** [comment]
Available in 
https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.4.0
https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v0.26.0

**Comment by umarali-nagoor** [comment]
Fix is available as part of terraform provider releases v1.4.0 and v0.26.0

https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.4.0
https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v0.26.0

Hence closing the issue.
